scripts/core/device.yml

```diff
@@ -393,7 +393,7 @@ etors:
     - name: GPU_HW_THREADS_PER_EU
       desc: "[uint32_t][optional-query] return Intel GPU number of threads per EU"
     - name: MAX_MEMORY_BANDWIDTH
-      desc: "[uint32_t][optional-query] return max memory bandwidth in Mb/s"
+      desc: "[uint64_t][optional-query] return max memory bandwidth in Mb/s"
```

Comment:
The spec for this says it should be in Mb/s, which I'd be surprised if we're overflowing (uint32 max mb would be around 4000 terabytes/second). Is the sycl RT expecting mb/s? Should we update it to b/s?

Reply:
I changed it to be b/s to match the sycl docs for this query in intel/llvm.

Reply:
I updated this to be capital B/s as it should be byte per second.